app/src/main/java/com/iron/espresso/presentation/home/study/StudyCreateActivity.kt

```diff
@@ -31,22 +28,14 @@ class StudyCreateActivity :
 
     private var localItem: LocalItem? = null
 
-    private val categoryImage by lazy {
-        intent.getIntExtra(KEY, 0)
-    }
-
     @SuppressLint("ClickableViewAccessibility")
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setToolbarTitle(TITLE)
         setNavigationIcon(R.drawable.ic_back_24)
 
         binding.run {
-            image.transitionName = categoryImage.toString()
-            image.load(categoryImage, true) {
-                supportPostponeEnterTransition()
-            }
-
+            category.text = intent.getStringExtra(STUDY_CATEGORY)
```

Comment:
```suggestion
            category.text = intent.getStringExtra(STUDY_CATEGORY).orEmpty()
```
혹시 null이면 'null' 이렇게 찍힐 것 같은?

Reply:
앗 그렇군요.. 바로 수정했습니다!
ff1733a